### PR TITLE
API Telemetry - Seventh Batch - AppInitialization, AppInstallDialog, AppWindow, VideoEffectsUtils (capabilities)

### PR DESCRIPTION
**Pull request by KangxuanYe**
## Description

Adding api names to compose api version tags for each API in `AppInitialization`, `AppInstallDialog`, `AppWindow` capabilities and update `videoEffectsUtil` and other files, which make apiVersionTag to be sent out in request to Host SDKS to make sure apiVersionTag could be added into telemetry event and then be passed to Kusto database.

For more details, please take a look on the description in first batch PR: https://github.com/OfficeDev/microsoft-teams-library-js/pull/1891

### Main changes in the PR:

1. Adding API names in telemetry files
2. update API call of each corresponding API to be able to send out request with api version tag.

### Related PRs: https://github.com/OfficeDev/microsoft-teams-library-js/pull/1891

### CODE REVIEW

File: change/@microsoft-teams-js-12adf1d8-9273-468e-ac29-c8839a34e6d9.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added apiVersionTag for telemetry in `AppInitialization`, `AppInstallDialog`, `AppWindow` capabilities and update `videoEffectsUtil` and other files.",
```
Comment (TrevorJoelHarris) [inline change/@microsoft-teams-js-12adf1d8-9273-468e-ac29-c8839a34e6d9.json:3]:
```suggestion
  "comment": "Added telemetry for `appInitialization`, `appInstallDialog`, `appWindow`, `calendar`, and `videoEffectEx` capabilities.",
```

**Review by TrevorJoelHarris — CHANGES_REQUESTED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

File: packages/teams-js/src/public/appInitialization.ts [hunk 2]
```diff
@@ -1,9 +1,20 @@
...
+  notifySuccessHelper,
+} from './app';
 
 /**
  * @deprecated
  * As of 2.0.0, please use {@link app} namespace instead.
+ *
+ * v2 APIs telemetry file: All of APIs in this capability file should send out API version v2 ONLY
  */
+const appInitializationTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_1;
```
Comment (AE-MS) [inline packages/teams-js/src/public/appInitialization.ts:16]: >V_1 [](http://example.com/codeflow?start=83&length=3)

How do I reconcile this being V1 with the comment above saying that it should send out V2 ONLY?

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: packages/teams-js/src/public/appInitialization.ts (hunk 2 above)
Comment (KangxuanYe) [reply packages/teams-js/src/public/appInitialization.ts:16]: This should be V1. Comment is incorrect. I need to fix it.

Comment (KangxuanYe) [reply packages/teams-js/src/public/appInitialization.ts:16]: Updated.

Comment (KangxuanYe) [reply packages/teams-js/src/public/appInitialization.ts:16]: Per comment on line 4-5, it is also easy to refer this is a V1 capability which started from TJS V1. The comment saying it is a V2 is a typo error.

**Review by TrevorJoelHarris — APPROVED** [review]

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**